ibis/file/csv/client.py

```diff
@@ -0,0 +1,68 @@
+import pandas as pd
+import ibis.expr.operations as ops
+from ibis.file.client import FileClient
+from ibis.file.utils import pathlib
+from ibis.pandas.core import data_preload, execute
+from ibis.pandas.client import pandas_dtypes_to_ibis_schema
+
+
+class CSVTable(ops.DatabaseTable):
+    pass
+
+
+class CSVClient(FileClient):
+    extension = 'csv'
+
+    def write_to_file(self, path, t, index=False, **kwargs):
```

Comment:
Can we model this as an `insert` operation? So, instead of `write_to_table` the API is this:

```python
con = ibis.csv.connect('...')
con.create_table('path/to/file.csv')  # touch path/to/file.csv
expr = ...
con.insert('file.csv', expr)  # calls execute(expr).to_csv(...)
```

Reply:
yep

Reply:
though this is a bit redundant, I would simply do

```
con = ibis.csv.connect('/path/to')
expr = ...
con.insert('file.csv', expr)  # calls execute(expr).to_csv(...)
```
so the difference from sql is that a ``con`` here effectively has state, namely the directory level. We could not have this and instead do this as part of the ``.insert`` operation itself.